data/sql_updates/candidate_history.sql

```diff
@@ -66,7 +66,8 @@ left join disclosure.cand_inactive inactive on
     fec_yr.cand_id = inactive.cand_id and
     fec_yr.fec_election_yr < inactive.election_yr
 left join staging.ref_pty ref_party on fec_yr.cand_pty_affiliation = ref_party.pty_cd
-where max_cycle >= :START_YEAR
+where max_cycle >= :START_YEAR and
+    fec_yr.cand_id not in (select distinct cmte_id from unverified_filers_vw where cmte_id similar to '(P|S|H)%')
```

Comment:
False alarm.  I didn't realize that the `cmte_id` is also `cand_id` in some instances.  This is good as is!